.github/workflows/lint.yml

```diff
@@ -25,3 +26,16 @@ jobs:
         with:
           # Keep same version as *.nix
           dprint-version: '0.36.1'
+
+  typos:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      # Keep same version as *.nix
+      - uses: crate-ci/typos@v1.14.10
+        with:
+          files: |
+            .
+            .github
+            .vscode
```

Comment:
https://github.com/crate-ci/typos/issues/779#issuecomment-1635761199
https://github.com/crate-ci/typos/blob/143cc59fab61e34b96188ba6745f902d7867bda0/docs/github-action.md
https://github.com/kachick/ruby-gem-template/actions/runs/5562681678/jobs/10161136530?pr=181

CI failed correctly, so I can understand the usage correctly.